tests/nested/core/hotplug/task.yaml

```diff
@@ -1,5 +1,7 @@
 summary: Create ubuntu core image, install snapd and test hotplug feature
 
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
```

Comment:
hmm should we have a TODO:UC20: for enabling this on UC20?

Reply:
I'll add that in a following PR, thanks for the review